haxelib/src/vshaxe/HaxelibExecutableConfiguration.hx

```diff
@@ -0,0 +1,12 @@
+package vshaxe;
+
+/**
+	Configuration for the Haxelib executable.
+	@since 2.23.0
+**/
+typedef HaxelibExecutableConfiguration = {
+	/**
+		Absolute path to the Haxelib executable, or a command / alias like `"haxelib"`.
+	**/
+	var executable(default, never):String;
+}
```

Comment:
This config could be a mere String, but with Haxelib exposed it is best IMHO to use a structure instead, like in `vshaxe.HaxeExecutableConfiguration`, because we avoid making a breaking change if we need more fields later.

Reply:
I agree, future-proofing it like that is good practice. 👍